### DISCUSSION TITLE
Python Noob...

### DISCUSSION THREAD

**Discussion by GrandizerGo** [Serene-Arc/bulk-downloader-for-reddit#459] [Q&A]
I am a Noob to python, I was however able to download, install, and run python. I then was able to download and install bdfr newest version.
I was surprised that there was no executable, but reading these comments I understand and that is fine.
I however do NOT know or understand how to take the old version 1.4.9 options I used and convert them to the proper syntax to use for the command line version of 2.2.0.
I use to be able to download all items from a certain user that I chose, now I do not know how to do so, or maybe I did, but I saw a message that said I had to be authenticated to do such? No idea how to do that.
I used to be able to download many of the items in an entire subreddit, however the reason I looked for the newer version is that the error file was basically not allowing me to download from certain sites like Redgif and others.
According to what I read on here, I should be able to grab those files from Redgif and a few others it says it supports as well.
Can anyone make two command lines that I could just edit myself to grab all posts from a user and the other to download all items from a subreddit?
I understand the no-dupes, and can add that myself, and I think I have the save to directory down as well.
Thank you.

**Comment by GrandizerGo** [comment]
17 days and no help?

**Reply by Serene-Arc** [reply]
Sorry for the wait but we do have other things to do, especially when it's clear that you haven't put in the effort to read the README on the front page of the repository. You can't expect to understand the tool completely without doing that. 

You say you don't know how to authenticate? Read the documentation and use the `--authenticate` option with the instructions provided. You say you don't know how to download from a specific user? Read the documentation and use the `--user` option. If you want to download from a subreddit, use the `--subreddit` option with the instructions provided.

If you are finding errors, it is most likely because the resource is no longer available on the website. The specifics of the error code will tell you why, I went to a great deal of effort to make sure that they were informative and verbose. If that is not the case, you are more than welcome to read the documentation and follow the steps to open a bug report so that I may investigate the issues further. 

There are example commands in the README to provide examples but it isn't our job to provide custom commands for every user. If you are still experiencing issues that you can't work out **after reading the documentation** then post your command here and we can help. But please, make an effort yourself first.

**Reply by GrandizerGo** [reply]
I did put in the effort, and as I did say, I learned how to install python and learn enough to install your program to even get it to work. And ran the updater a few time in case I had something wrong, 
However I am not a programmer and never claimed to be, nor have I ever said I did not read the instructions, in fact someone I work with who does in fact write python read it and was also not able to answer the questions I had either, and he was offered payment as well.  
You have no problem with it as you wrote it. What you find easy is not so easy for someone else.
The fact that the commands as listed in your readme do not work on a first install of Python is proof of that and took a long time to figure out why the commands did not work. If you ONLY have version 3 of Python you get an error when running a command that starts with python3 as all of the examples in your documentation do. 
But I am supposed to know that never having learned Python. It was with my buddies help that finally we figured out the need to remove the number 3 from the commands you use.

**Reply by Serene-Arc** [reply]
That is not the norm on most systems and you did not post any commands or errors. You just said "I don't know how to do this" without reading the documentation. There are instructions on how to authenticate and so on in the README. Since you have now, then there is no further need for this thread.
